function addTwoNumbers(a, b)
{
    try{
        console.log(a + b);
    }catch(err){
        console.log(err);
    }
    
}

addTwoNumbers(5, "5");